rstudio/c9s-python-3.9/Dockerfile

```diff
@@ -50,9 +50,12 @@ COPY rsession.conf /etc/rstudio/rsession.conf
 # package installation 
 RUN dnf install -y libsodium-devel.x86_64 libgit2-devel.x86_64 libcurl-devel harfbuzz-devel.x86_64 fribidi-devel.x86_64 cmake "flexiblas-*" \
     && dnf clean all && rm -rf /var/cache/yum
-RUN R -e "install.packages('Rcpp')"
```

Comment:
Why this line is been deleted?

Reply:
We have decided to add it to the install.versions to be able to version lock this package.

Reply:
Have in mind this should be installed before any other package